sim/mcuboot-sys/src/c.rs

```diff
@@ -60,5 +71,9 @@ mod raw {
 
         pub static BOOT_MAGIC_SZ: u32;
         pub static BOOT_MAX_ALIGN: u32;
-    }
+
+        pub fn ecdsa256_sign_(privkey: *const u8, hash: *const u8,
+                              hash_len: libc::c_uint,
+                              signature: *mut u8) -> libc::c_int;
+}
```

Comment:
This brace should be indented to line up with the `extern "C" {` above, so 4 spaces.

Reply:
fixed